package.json

```diff
@@ -2778,17 +2778,17 @@
         "vscode-codicons": "0.0.13",
         "vscode-nls-dev": "^3.3.2",
         "vscode-test": "^1.4.1",
-        "webpack": "^4.44.2",
+        "webpack": "^4.46.0",
         "webpack-cli": "^3.3.12"
     },
     "dependencies": {
         "@azure/arm-appservice": "^6.1.0",
         "@azure/arm-containerregistry": "^8.0.0",
         "@azure/storage-blob": "^12.3.0",
         "@docker/sdk": "^1.0.3",
-        "@grpc/grpc-js": "^1.2.2",
-        "dayjs": "^1.10.2",
-        "dockerfile-language-server-nodejs": "^0.2.1",
+        "@grpc/grpc-js": "^1.2.3",
+        "dayjs": "^1.10.3",
+        "dockerfile-language-server-nodejs": "^0.2.2",
```

Comment:
Hopefully this will help with #2601.

Reply:
@rcjsuen Should we just grab 0.2.2 for our 1.9.1 release (~Monday)?

Reply:
@bwateratmsft It shouldn't hurt. The actual code changes are [minimal](https://github.com/rcjsuen/dockerfile-language-server-nodejs/compare/v0.2.1...v0.2.2), just implementing what was suggested in https://github.com/microsoft/vscode-docker/issues/2601#issuecomment-756134327.

Has anyone else hit #2601? The stack trace indicates it was reproduced on 1.9.5-alpha. 🤔

Reply:
I've got it showing up on our 1.9.1 candidate build. I'll make a PR and we can discuss in our team triage today.